packages/angular/projects/angular/src/components/radio-group/radio-group.component.ts

```diff
@@ -31,16 +29,16 @@ export type RadioGroupInputs<T extends RadioGroupValue> =
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class RadioGroupComponent<T extends RadioGroupValue>
-implements OnInit, RadioGroupInputs<T> {
+  extends RadioGroupBaseComponent<T>
+  implements OnInit, RadioGroupInputs<T> {
   private inputValue?: T;
 
-  @Input() set value(value: T) {
+  @Input() set value(value: T | undefined) {
```

Comment:
No, we cannot make ```value`` optional parameter because if we do it we got the next ts error:
```
TS1051: A 'set' accessor cannot have an optional parameter.
```